libs/common/src/lib/resources/edit-resource/annotation/paragraph-annotation/paragraph-annotation.component.ts

```diff
@@ -300,4 +301,36 @@ export class ParagraphAnnotationComponent extends SelectFirstFieldDirective impl
       ),
     );
   }
+
+  @HostListener('window:keyup', ['$event'])
+  private onKeyUp(event: KeyboardEvent) {
+    if (document.activeElement !== document.body) return;
+    forkJoin([this.paragraphs.pipe(take(1)), this.isAdminOrContrib.pipe(take(1))]).subscribe(
```

Comment:
this seems a bit heavy to me as you will do this call on each keyup event, while you need it only on Enter, Delete and Escape. It would be better to check the event key first and act on it only on the events of interest, no?

Reply:
let me rethink it

Reply:
Fixed